.github/workflows/go-ci.yaml

```diff
@@ -14,75 +14,82 @@ jobs:
     needs: get_refs
     strategy:
       matrix:
-        go-version: [1.17.x, 1.18.x]
+        go-version: [1.17.x, 1.20.x]
```

Comment:
Shouldn't we bump the lower one too? 

> Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release.

Reply:
Sure, we can remove 1.17/1.18 support now since we're doing a major bump anyway